manifests/t/Tutanota/Tutanota/259.241213.0/Tutanota.Tutanota.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Tutanota.Tutanota
+PackageVersion: 259.241213.0
+InstallerLocale: en-US
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+ReleaseDate: 2024-12-16
+Installers:
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://github.com/tutao/tutanota/releases/download/tutanota-desktop-release-259.241213.0/tutanota-desktop-win.exe
+  InstallerSha256: FC3DBE41AEE2E73C7F63CC1D0CCE3225169ADE6633A938F5B98AFB294CD47BC3
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x86
+  Scope: machine

```

```suggestion
- Architecture: x64
  Scope: machine
  ElevationRequirement: elevatesSelf
```
When installing to the machine scope, this installer requires administrator privileges.